spec/wikidata/diff/api_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# spec/wikidata/diff/api_spec.rb
+
+require './lib/wikidata/diff/api'
+require 'rspec'
+
+RSpec.describe Api do
+    describe '.get_revision_contents' do
```

Comment:
I think this pair of tests is basically doing what we want now, but they should be refactored to be more idiomatic.

The `describe` block here indicates a particular method that is supposed to be tested, but the first test within it doesn't use that method. It should be in a different `describe` block.